doc/user/content/release-notes.md

```diff
@@ -105,27 +105,33 @@ List new features before bug fixes.
 
 {{% version-header v0.12.0 %}}
 
-- Support [`PREPARE`](/sql/prepare/), [`EXECUTE`](/sql/execute/), and [`DEALLOCATE`](/sql/deallocate/)
-for SQL statements.
+- Allow including the message partition, offset, and timestamp in [Kafka
```

Comment:
Just for clarity: are headers what's missing to round off #1577?

Reply:
style nit:

> Let [Kafka sources](/sql/create-source/avro-kafka/) automatically include columns with messages' partition, offset, and timestamp via the new `INCLUDE PARTITION`, `INCLUDE OFFSET`, and `INCLUDE TIMESTAMP` options, respectively.

or something. The verb verb structure here isn't my favorite.